### ISSUE TITLE
Cannot create initial config file in Windows

### ISSUE THREAD

**Issue by obenshaindw** [DCAN-Labs/nda-abcd-s3-downloader#4]
After following the readme and help documentation provided with the tool, the script is unable to create or locate the config file.

```
c:\Users\obenshaindw\Documents\Code\Python\nda-abcd-s3-downloader>python download.py -i datastructure_manifest.txt -o c:\Users\obenshaindw\Downloads
Derivatives downloader called at 2020:01:04 00:15 with:
Traceback (most recent call last):
  File "download.py", line 379, in <module>
    _cli()
  File "download.py", line 108, in _cli
    make_nda_token(args.credentials)
  File "download.py", line 309, in make_nda_token
    password
  File "C:\Python36\lib\subprocess.py", line 267, in call
    with Popen(*popenargs, **kwargs) as p:
  File "C:\Python36\lib\subprocess.py", line 709, in __init__
    restore_signals, start_new_session)
  File "C:\Python36\lib\subprocess.py", line 997, in _execute_child
    startupinfo)
FileNotFoundError: [WinError 2] The system cannot find the file specified
```
If I specify a file with the -c command, the file is created successfully.

**Comment by drewwint** [comment]
I am running into the same issue. However, this issue persists even when I specify the file's location using the -c flag suggested above. Is there another workaround?


![error](https://user-images.githubusercontent.com/53614162/178043644-0a0f62ed-0993-46ed-9f3c-b1d79cb4ed26.png)

**Comment by drewwint** [comment]
Yes I'm sure - I just triple checked it again using multiple methods of creating the config file and -c flagging the file location with no luck.

**Comment by ericearl** [comment]
@drewwint Have you used the command `aws configure` once on the server you're trying this on?  I'm wondering if the AWS files are present that this script relies on appending to...  I might have some time later this week to get on a call with you and then we can post the solution back here?  Have you tried the command on a non-Windows OS yet (if that's an option)?

@ericfeczko It looks like @drewwint is trying this in a Windows Powershell.

**Comment by drewwint** [comment]
I appreciate it - I sent you an email @ericearl

**Comment by drewwint** [comment]
@ericearl and I found that when using Windows PowerShell, we found that editing line 314 in the download.py script from "python3" to just "python" resolved this specific issue. 

Although this may be able to be done with PowerShell I found using the linux subsystem in windows is much more straightforward and it worked seamlessly. Highly recommend downloading WSL and using the linux subsystem when running in windows.
